vuepress/README.md

```diff
@@ -3,7 +3,7 @@ home: true
 heroImage: ./vue-i18n-logo.png
 actionText: Get Started →
 actionLink: introduction.md
-footer: MIT Licensed | Copyright © 2018 kazuya kawaguchi
+footer: MIT Licensed | Copyright © 2019 kazuya kawaguchi
```

Comment:
Thanks for opening this PR.
Could you also update copyright year in other files ?
E.g. https://github.com/kazupon/vue-i18n/blob/ccf29f88d343e9baf2e6a5a9c04ad992c4f5906d/vuepress/zh/README.md

Reply:
Yeah Sure!